examples/binding_affinity_prediction/utils.py

```diff
@@ -37,29 +55,54 @@ def load_dataset(args):
         Input arguments.
     Returns
     -------
-    dataset
-        Full dataset.
     train_set
         Train subset of the dataset.
     val_set
         Validation subset of the dataset.
+    test_set
+        Test subset of the dataset.
     """
     assert args['dataset'] in ['PDBBind'], 'Unexpected dataset {}'.format(args['dataset'])
     if args['dataset'] == 'PDBBind':
-        dataset = PDBBind(subset=args['subset'],
-                          load_binding_pocket=args['load_binding_pocket'],
-                          zero_padding=True)
-        # No validation set is used and frac_val = 0.
-        if args['split'] == 'random':
-            train_set, _, test_set = RandomSplitter.train_val_test_split(
+        if args['model'] == 'PotentialNet': 
+            from functools import partial
+            from dgllife.utils import PN_graph_construction_featurization
```

Comment:
Put all lines of code for importing together, i.e. moving them to L17-L18.

Reply:
fixed